protoc-options.txt

```diff
@@ -2,3 +2,4 @@
 --proto_path=./cosmos-sdk/proto
 --proto_path=./cosmos-sdk/third_party/proto
 --proto_path=./ibc-go/proto
+--proto_path=./third_party
```

Comment:
Do we need this to be in the `third_party` directory because it's imported that way? If not, could we move it to the root directory (`./google/protobuf`)?

I think this could be a submodule too: https://github.com/protocolbuffers/protobuf/tree/main/src/google/protobuf

Reply:
We won't need this once https://figmentio.atlassian.net/browse/GRAPH-384 is done. I would suggest to merge this as is I'll remove it once the others ticket is done.

Reply:
Do you think you can finish GRAPH-384 today? If so, we can merge this as-is.

We have to release this package early next week and I don't want to mix two different approaches (submodule vs. copy-pasting) in the released version.

Reply:
GRAPH-384 is almost completed. Will push the change on this branch.